dailiza-bot/text_patterns.py

```diff
@@ -3,14 +3,56 @@
 """
 
 psychobabble = [
-    [r"geht.{0,5}s.{0,5}dir",
+    [r"[Gg]eht.{0,5}s.{0,5}dir",
     ["Danke. Mir geht es gut und dir?",
     "Sehr gut, danke. Und wie läuft's bei dir?",
     "Ich kann nicht klagen. Was ist mit dir?"]],
 
-    [r"Ich brauche (.*)",
+    [r"[Ii]ch brauche (.*)",
     ["Warum brauchst du {0}?",
     "Würde {0} dir denn wirklich helfen?",
     "Bist du sicher, dass du {0} brauchst?"]],
 
+    [r"[Ii]ch kann (.*) nicht",
+    ["Wie kommst du darauf, dass du {0} nicht kannst",
+    "Vielleicht wirst du es schaffen, wenn du {0} versuchst",
+    "{0} zu können ist meiner Meinung nach nicht so wichtig"]],
+    
+    [r"[Ww]ie (.*)",
+    ["Vieleicht kannst du das googlen?",
+    "Ich kann dir dabei leider nicht helfen",
+    "Ich möchte mich dazu nicht äußern"]],
+    
+    [r"[Ww]arum (.*)",
+    ["Ich kenne mich damit nicht aus",
+    "Weil Google dir dabei besser helfen kann",
+    "Das weiß ich nicht"]],
+    
+    [r"[Jj]a",
+    ["Das finde ich sehr schön",
+    "Cool",
+    "Ich freue mich"]],
+    
+    [r"[Nn]ein",
+    ["Das finde ich nicht schön",
+    "Ohh Sorryy",
+    "Das tut mir leid"]],
+
+    [r"[bB]ot (.*)",
+    ["Meinst du mich?",
+    "Findest du es komisch, mit mir zu chatten?",
+    "Machst das dich glücklich, mit mir zu chatten?"]],
+
+    [r"(.*)",
```

Comment:
(.*) would include all possible combinations of words so dailiza will have a somewhat appropriate answer to unrecognized text patterns but (.*) would also include "exit". This means it will give responses to exit which arent appropriate. I suggest recognizing "exit" as a text pattern in the psychobabble and adding responses such as "bye","goodbye :)" or something.

Reply:
Alternatively, one could make the following changes to the run_dailiza_bot() function in the dailiza_bot.py file

def run_dailiza_bot():
    user_input = ""
    while "exit" not in user_input:
        user_input = input(">> ")
        if "Bye".lower() in user_input.lower():
            print("Bye ;)")
            break
        if "exit" in user_input.lower():
            break
        print(dailiza_answer(user_input))